docs/chapters/04_features.md

```diff
@@ -1,5 +1,34 @@
 # Features
 
+## Local Provider
+
+All Booster projects come with a local development environment configured by default, so you can test your app before deploying it to the cloud.
+
+You can see the configured local environment in your `src/config/config.ts` file:
+
+```typescript
+Booster.configure('local', (config: BoosterConfig): void => {
+  config.appName = 'my-store'
+  config.providerPackage = '@boostercloud/framework-provider-local'
+})
+```
+
+Note how, contrary to the rest of the environments, the configuration of the local environment is using the `config.providerPackage` field,
+and not the `config.provider`.
+
+This is a special field that allows deferring the loading of the
+provider to later, so you don't get dependencies errors when you
+upload your app to production, given that the local provider lives in
+the `devDependencies` section of your `package.json`
```

Comment:
@NickSeagull 
🤔  I think we should bet on only one option. To be honest, I like your solution of providing the package name and the rockets as separate properties. After thinking about it, I don't see drawbacks. 

So I would write in the documentation only this way of specifying the provider and the rockets. Besides, I would rewrite all the configuration environments we have in the integration tests, etc. to the new way and mark the `provider` property as deprecated with a message.

Do you see any inconvenience in doing this? Any possible design flaw?

Reply:
No, my decision was more geared towards backwards compatibility, but I would just get one as well